content/contributing/opencue/Getting started/getting-started-linux.md

```diff
@@ -157,9 +157,13 @@ To build and run it with IntelliJ IDEA:
    IntelliJ downloads Gradle and all source dependencies, then compiles the project and
    runs tests.
 
-1. Browse to the `src/main/java/com.imageworks/spcue/CuebotApplication` file.
+1. To setup run configurations go to **Run** > **Edit Configurations**
```

Comment:
Could you make the same changes to the Mac version of this page please? I've check on my local machine and the steps are the same.

Reply:
Added in https://github.com/AcademySoftwareFoundation/opencue.io/pull/277/commits/c3714890fa9f45d5ec2aa4d194aadabcea0754f7